### ISSUE TITLE
Cannot use KeeperCommander with corporate SSO

### ISSUE THREAD

**Issue by sndrr** [Keeper-Security/Commander#337]
### Actual situation
Trying to log in with KeeperCommander on Linux fails. My user has a corporate account for the company `EXAMPLE` and a keeper login as `user@example.com`. Our Keeper instance is in the EU.

```
$ keeper --server keepersecurity.eu --user user@example.com --debug login
INFO:root:Debug ON
DEBUG:root:Params: <keepercommander.params.KeeperParams object at 0x7fdbd81c5d90>
This account requires 2FA Authentication
     U2F (FIDO Security Key)
  2. Send SMS Code [ ENABLED ]
     TOTP (Google Authenticator)
     DUO
Selection: 2

Enter 2FA Code (use code sent to): 012345
Password: 
ERROR:root:Account validation error.
Your Keeper session needs to be refreshed. Please logout and login again.
...      User(Email): user@example.com
This account requires 2FA Authentication
     U2F (FIDO Security Key)
  2. Send SMS Code [ ENABLED ]
     TOTP (Google Authenticator)
     DUO
Selection: 2

Enter 2FA Code (use code sent to): 123456
Password: 
ERROR:root:Communication Error: Account validation error.
Your Keeper session needs to be refreshed. Please logout and login again.
Not logged in> 
```
And so on.  It seems that it cannot communicate with the corporate SSO server, as the Linux Electron Desktop client does (in a separate window).

### Expected situation
I can login with my corporate account.

### Steps to reproduce
1. Sign up for a (corporate) Keeper account with SSO
2. Set up 2FA in the web client
3. Download and install KeeperCommander with pip
4. Try to log in with CLI
5. Approve the login with KeeperCommander (or enter the validation code)
6. Observe that you end up in an endless loop (2FA > Account validation error > 2FA > Account validation error > 2FA ..)

### Environment
* KeeperCommander 4.70
* Python 3.8.5
* OS: Linux

**Comment by maksimu** [comment]
Hello @sndrr

Have you tried to login to the same account via Web UI with the same password that you are using when tried to login to Commander?

Please reach out to me directly and we can try to resolve this issue together. My Email: maksim {AT] keepersecurity.com

**Comment by maksimu** [comment]
Also, for the SSO login, you will need to enable Alternate Master Password in your account  in order to login. See "[SSO Accounts](https://github.com/Keeper-Security/Commander#sso-accounts)" section in readme

**Comment by sndrr** [comment]
@maksimu Yes, both the web client and desktop client work. Unfortunately I'm unable to set the `Alternate Master Password`, because of company policy (I don't have the privileges to do so). I'll reach out by mail.

**Comment by sndrr** [comment]
Ah, I see it's stated in the README that [SSO is not yet supported](https://github.com/Keeper-Security/Commander/tree/record-types#sso-accounts).
